pkgs/shells/bash/5.nix

```diff
@@ -107,6 +107,8 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  makeFlags = [ ];
```

Comment:
Please don't make the code more complicated just to avoid a mass rebuild.  Mass rebuilds are fine.  The bash changes can do to staging separately from the rest of this if you want.